### PR TITLE
Relates to CAMEL-14910: Allow specific manually maintained attributes in component adoc headers

### PR DESCRIPTION
**Pull request by djencks**
AemieJ's work on organizing the component files into groups needs a little help so the `:group:` attribute doesn't get overwritten by the generation process.

This adds a hardcoded list of attribute names whereby attributes by those names will be preserved through the generation process, moved to the end of the list of attributes.

### CODE REVIEW

**Comment by davsclaus** [comment]
Can you fix CS by running

mvn compile -P sourcecheck

**Comment by djencks** [comment]
I tried running `mvn compile -P sourcecheck` in the plugin directory and in tooling and saw no problems, so I fixed things based on the GitHub report.  What's the magic to get this to work?

**Comment by djencks** [comment]
Although source check still fails at GitHub, it looks to me as if it's a pre-existing problem not in the code I modified.

**Comment by djencks** [comment]
Davsclaus This doesn't do much by itself but is needed for AemieJ's work. Would it be OK for me (or you) to merge it in, I think it will make the following work easier to manage?  Thanks

File: tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/UpdateReadmeMojo.java [hunk 1]
```diff
@@ -20,13 +20,9 @@
...
 import java.io.InputStream;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Objects;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
```
Comment (davsclaus) [inline tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/UpdateReadmeMojo.java:23]: * imports are usually not valid

Comment (djencks) [reply tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/UpdateReadmeMojo.java]: Should be fixed now.... If this is an important rule, why doesn't sourcecheck find the problem?

**Pull request merged**